CMakeLists.txt

```diff
@@ -525,6 +527,22 @@ IF(ASPECT_WITH_LIBDAP)
   ENDIF()
 ENDIF()
 
+
+# Check for FastScape library and link it to ASPECT if requested
+SET(ASPECT_WITH_FASTSCAPE OFF CACHE BOOL "Check if the user wants to compile ASPECT with the landscape evolution code FastScape.")
+MESSAGE(STATUS "Using ASPECT_WITH_FASTSCAPE = '${ASPECT_WITH_FASTSCAPE}'")
+IF(ASPECT_WITH_FASTSCAPE)
+  IF (EXISTS ${FASTSCAPE_DIR}/libfastscapelib_fortran.so)
+     MESSAGE(STATUS "FastScape library found at ${FASTSCAPE_DIR}/libfastscapelib_fortran.so")
+     FIND_LIBRARY(FASTSCAPE NAMES fastscapelib_fortran PATHS $ENV{FASTSCAPE_DIR} ${FASTSCAPE_DIR} PATH_SUFFIXES lib NO_DEFAULT_PATH)
```

Comment:
Should we include a minimum version here, like we do for deal.II above? Just to make sure no one tries to build this with very old Fastscape versions that may not be compatible.